assets/css/madetech-frontend.css

```diff
@@ -8910,4 +8910,13 @@ blockquote::before {
   margin-left: 50px;
 }
 
+.identity-section {
```

Comment:
Let's avoid adding custom styles. Could this style have been applied using a bootstrap utility class?